src/components/30-organisms/modal/story.js

```diff
@@ -56,6 +69,9 @@ export const Modal = () => {
         days, their intelligence quotient begins to decline. After three weeks
         hospitalization, it can already be 20% lower than usual.
       </axa-text>
+      <axa-button class="modal-story__close-modal-child-button">
+        I accept the terms and conditions
+      </axa-button>
     </axa-modal>
 
     <button
```

Comment:
do we need this button here for tests? because we have knobs and dont really need this button here.